scratch/evaluate_four_elements_traffic.cc

```diff
@@ -67,16 +138,44 @@ main (int argc, char *argv[])
                                                      interfaces.GetAddress (0), 9);
 
   ApplicationContainer serverApps = gamingStreamingHelper.Install (nodes.Get (1));
+  gamingServer = StaticCast<GamingStreamingServer> (serverApps.Get (0));
   serverApps.Start (Seconds (1.0));
-  serverApps.Stop (Seconds (10.0));
+  serverApps.Stop (Seconds (11.0));
   /** End of Streaming Server*/
 
   PacketSinkHelper sink ("ns3::UdpSocketFactory", InetSocketAddress (Ipv4Address::GetAny (), 9));
   ApplicationContainer sinkApps = sink.Install (nodes.Get (0));
+  packetSink = StaticCast<PacketSink> (sinkApps.Get (0));
   sinkApps.Start (Seconds (0.0));
-  sinkApps.Stop (Seconds (11.0));
-
+  sinkApps.Stop (Seconds (12.0));
+  if (throughput)
+    {
+      Simulator::Schedule (Seconds (1.1), &CalculateThroughput);
+    }
+  packetSink->TraceConnectWithoutContext ("Rx", MakeCallback (&ReceivePacket));
+
+  Simulator::Stop (Seconds (12.1));
```

Comment:
Same comment as for Crazy taxi. ServerApp and sinkApp stop time should be also changed accordingly